source/mongodb-handshake/handshake.rst

```diff
@@ -52,22 +69,40 @@ Specification
 Connection handshake
 --------------------
 
-The ``isMaster`` handshake MUST be performed on every socket to any and all servers
+MongoDB uses the ``hello`` or ``isMaster`` commands for handshakes and topology
+monitoring. ``hello`` is the modern and preferred command. ``isMaster`` is referred
+to as "legacy hello" and is maintained for backwards compatibility with servers
```

Comment:
With the terminology introduced here, I'd vote for replacing ``isMaster`` with `legacy hello` in the "Backwards Compatibility" section and "Q&A".

Reply:
Fair point. Done.